### DISCUSSION TITLE
Feature Request: Improve Pinned Tab Behavior

### DISCUSSION THREAD

**Discussion by kristijanribaric** [zen-browser/desktop#1453] [Ideas]
### Current Issue
When a tab is pinned, it retains the URL it was left on rather than the one it was originally pinned to. This can be inconvenient when reopening pinned tabs, as users may expect them to revert to the initial pinned URL.

### Proposed Solution
When a pinned tab is closed using a shortcut like `CTRL + W`, reopening the tab should reset the URL to the one it was originally pinned to, not the most recently visited URL.

**Example:**  
Arc Browser implements this feature well by resetting the URL of pinned tabs to their initial state upon reopening. Mimicking this behavior would enhance the user experience.

### Implementation Suggestion
- When a tab is pinned, save it as a bookmark in a folder labeled `"Pins - [Workspace Name]"`.
- Users should have the option to pin tabs across all workspaces. In this case, the tab would be saved in a general `"Pins"` bookmark folder.
- **Advantages of this approach:**
  - Pinned tabs can be accessed on mobile devices by navigating to the bookmark folder.
  - Pinned tabs could (potentially) be synced across multiple browsers.
  - Pinned tabs would essentially function as bookmarks, visible only on the sidebar.

How you would be able to see pinned tabs on mobile:
![firefox_mobile_1](https://github.com/user-attachments/assets/5dafa2aa-cc94-4407-b68f-3cd7a2abaec1)

### Alternative Solution
Keep the current behavior, where pinned tabs retain the last visited URL, but visually differentiate them by adding a separator between pinned tabs and regular tabs. In this case, the existing pinned tabs can remain as they are, while the new behavior (resetting the URL to the original one) could be labeled as **"Favorites"**.

Picture of the idea:
![tabs](https://github.com/user-attachments/assets/f50f1218-6b16-4513-94b5-ffa75b339936)

**Comment by StudioMovieGirl** [comment]
this mad lad actually added some of this
